pkg/organizations/client.go

```diff
@@ -130,21 +136,9 @@ type ListOrganizationsResponse struct {
 	ListMetadata common.ListMetadata `json:"listMetadata"`
 }
 
-type OrganizationDomainDataState string
-
-const (
-	Verified OrganizationDomainDataState = "verified"
-	Pending  OrganizationDomainDataState = "pending"
-)
+type OrganizationDomainDataState = organization_domains.OrganizationDomainDataState
```

Comment:
I wonder if constant values like this have to be redeclared unfortunately. I don't think they get picked up when defining a type alias.